paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/PaymentSheetCustomerSessionTest.kt

```diff
@@ -163,4 +251,31 @@ class PaymentSheetCustomerSessionTest {
             )
         }
     }
+
+    private fun createReplacements(
+        isSaveEnabled: Boolean,
+        allowRedisplayOverride: String?
+    ): List<Replacement> {
+        val replacements = mutableListOf<Replacement>()
+
+        if (!isSaveEnabled) {
+            replacements.add(
+                Replacement(
+                    original = "\"payment_method_save\": \"enabled\"",
+                    new = "\"payment_method_save\": \"disabled\"",
+                )
+            )
+        }
+
+        allowRedisplayOverride?.let {
+            replacements.add(
+                Replacement(
+                    original = "\"payment_method_save_allow_redisplay_override\": null",
+                    new = "\"payment_method_save_allow_redisplay_override\": \"$it\"",
+                )
+            )
+        }
+
+        return replacements
```

Comment:
I figured we could just replace these lines in the files rather than introduce new ones with one line changes. Though open to just making those element sessions responses if we feel like we should just have a full elements session response.

Reply:
Seems reasonable to me. Can we remove any mock responses because of this?

Reply:
At least right now no. But will help prevent additive changes.